master/static/srv/get_model_versions.sql

```diff
@@ -36,3 +36,5 @@ SELECT
     version AS version,
     creation_time
```

Comment:
Maybe
```suggestion
    mv.version,
    m.creation_time
```
for clarity.

Reply:
Ended up changing `m.creation_time -> mv.creation_time` since the proto docs indicate that is what should be there.